WikipediaUnitTests/Code/QuadKeyTests.swift

```diff
@@ -0,0 +1,57 @@
+import XCTest
+@testable import Wikipedia
+
+class QuadKeyTests: XCTestCase {
+
```

Comment:
Added comments to this and QuadKey.swift - let me know if they're helpful or could be clarified or expanded